3ds Max/Max2Babylon/Exporter/AnimationGroup.cs

```diff
@@ -522,6 +525,66 @@ public static void LoadDataFromContainers()
             }
         }
 
+        
+        private static void ResolveMultipleInheritedContainer(IIContainerObject container)
+        {
+            string helperPropBuffer = string.Empty;
+            container.BabylonContainerHelper().GetUserPropBuffer(ref helperPropBuffer);
+
+            List<IINode> containerHierarchy = new List<IINode>() { container.ContainerNode };
+            containerHierarchy.AddRange(container.ContainerNode.NodeTree());
+
+            //manage multiple containers inherithed from the same source
+            foreach (IINode n in containerHierarchy)
+            {
+                //change the guid of the node
+                //replace the guid in the babylon helper
+                string oldGuid = n.GetStringProperty("babylonjs_GUID",Guid.NewGuid().ToString());
+                n.DeleteProperty("babylonjs_GUID");
+                Guid newGuid = n.GetGuid();
+                helperPropBuffer = helperPropBuffer.Replace(oldGuid, newGuid.ToString());
+
+            }
+
+            //replace animationList guid to have distinct list of AnimationGroup for each container
+            string animationListStr = string.Empty;
+            container.BabylonContainerHelper().GetUserPropString(s_AnimationListPropertyName, ref animationListStr);
+            string[] animationGroupGuid = animationListStr.Split(AnimationGroup.s_PropertySeparator);
+            foreach (string guidStr in animationGroupGuid)
+            {
+                Guid newAnimGroupGuid = Guid.NewGuid();
+                helperPropBuffer = helperPropBuffer.Replace(guidStr, newAnimGroupGuid.ToString());
```

Comment:
Looks like my test scene is causing a null ref here where we retrieve an empty animationList string.